store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/key/core/AbstractCoreKeyAccumuloElementConverter.java

```diff
@@ -439,7 +453,7 @@ protected void addPropertiesToElement(final Element element, final Key key) {
                 getPropertiesFromTimestamp(element.getGroup(), key.getTimestamp()));
     }
 
-    protected Edge getEdgeFromKey(final Key key, final Map<String, String> options) {
+    private Edge getEdgeFromKey(final Key key, final Map<String, String> options) {
```

Comment:
Please can we keep these methods as protected. This class can be extended so it would be useful to allow child classes to be able to call these methods if they need to.